docs/changeset-guide.md

```diff
@@ -0,0 +1,132 @@
+# Changeset Creation Guide
+
+## When to Create a Changeset
+
+### 1. Identify Affected Packages
+
+Determine which packages are affected by your changes.
+
+### 2. Changeset Required When
+
+All of the following conditions must be met:
+
+1. **Target packages** (not in ignore list):
+   - `@liam-hq/cli`
+   - `@liam-hq/erd-core`
+   - `@liam-hq/db-structure`
+   - `@liam-hq/ui`
+
+2. **User-facing changes**:
+   - New features
+   - Bug fixes
+   - API changes
+   - Performance improvements
+   - Behavioral changes
+
+### 3. Changeset Not Required
+
+- **Ignored packages**:
+  ```
+  @liam-hq/agent, @liam-hq/app, @liam-hq/docs,
+  @liam-hq/figma-to-css-variables, @liam-hq/db,
+  @liam-hq/jobs, @liam-hq/storybook, @liam-hq/github,
+  @liam-hq/schema-bench
+  ```
+
+- **Non-user-facing changes**:
+  - Internal refactoring (no external API changes)
+  - Test additions/modifications only
+  - Development tooling changes
+  - Comments or documentation only
+  - Internal type definition improvements
+
+## How to Create a Changeset
+
+### 1. Determine Version Type
+
+- **major**: Breaking changes
+- **minor**: New features (backward compatible)
+- **patch**: Bug fixes, small improvements
+
+### 2. Create Changeset File
+
+```
+.changeset/[descriptive-change-name].md
+```
+
+### 3. Format
+
+```markdown
+---
+"@liam-hq/package-name": version-type(minor|patch)
+---
+
+[emoji] Brief description
+
+- Specific change details
+- Technical details (if necessary)
+```
```

Comment:
I noticed that there is a problem with the loose list, so I will adjust the format from now on.

ref: https://github.com/liam-hq/liam/pull/2421#discussion_r2193632706